packages/cli/lib/lang.js

```diff
@@ -52,7 +52,7 @@ const getTextValue = lookupDotNotation => {
       previouslyCheckedProp = prop;
     });
   } catch (e) {
-    logger.error(
+    logger.debug(
```

Comment:
I changed this to log only in debug mode but let me know if you'd rather keep it as error. I figure that the end user may not want to see missing translation logs, but I can also see us missing some of these missing translation warnings while working in the CLI

Reply:
Hmm that's a tricky one. My gut feeling is that this will _probably_ lead to us missing more errors - personally I don't always test things with the `--debug` flag.

Reply:
:two-cents: I'm ok with keeping this as `debug`, since this isn't an error that will break the actual command.

Reply:
Yeah I'm seeing both sides here. I agree that it's pretty gross for us to show the user the long `Unable to access language property some.long.key.string. Failed to access...` error message. But I think we'd end up missing a ton of errors if we don't log anything by default, just because it can be tough to notice the absence of a log.

I wonder if we could split the difference. What if we have two different versions of what gets logged here? By default we could log something that doesn't include the lang key and is much shorter.

**For example:**

With `--debug` (log what we log now)
`[Debug] Unable to access language property some.long.key.string. Failed to access...`

Without `--debug` (log a condensed version)
`[Error] Unable to access language property.`

Reply:
Also something to remember is even in the HubSpot app we will show the "[missing translation en.something.blah]" strings to users (that's not great either tbh. Just making the point that it's not a new thing)

Reply:
Yeah good points there. I think the two log approach is a good happy medium, less gross to the end user but will still be shown as an error. Will push an update shortly